x/epochs/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=cosmos-sdk-epochs
```

Comment:
Great! Added in sonarcloud. Can you follow as well the rest of this list: https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#gomod?

Reply:
Do we need a vanity pr?

Reply:
Yes